packages/compass-e2e-tests/helpers/commands/wait-for-connection-result.ts

```diff
@@ -18,6 +18,8 @@ export async function waitForConnectionResult(
     // indicator that we are connected to the server
     selector = TEST_COMPASS_WEB
       ? '[data-testid="workspace-tab-button"][title=Databases]'
+      : TEST_MULTIPLE_CONNECTIONS
+      ? Selectors.SidebarTreeItems
```

Comment:
This kinda assumes that any connection is good enough, but you could have already had a connection before you even connected, so then it actually won't wait for anything. Ideally we'd wait for the element for the expected connection name to appear, but we don't have that context here.